src/theory/arith/linear/constraint_forward.h

```diff
@@ -27,6 +27,9 @@
 #include "cvc5_private.h"
 
 namespace cvc5::internal {
+
+class Rational;
```

Comment:
Could also use a direct `#include "util/rational.h"` here, either works. Let me know if you'd prefer a change.